tests/functional/ext/docparams/parameter/missing_param_doc_required_Google.py

```diff
@@ -16,3 +16,18 @@ def test_multi_line_parameters(param: int) -> None:
             a description
     """
     print(param)
+
+
+def test_missing_func_params_in_google_docstring(  # [missing-param-doc, missing-type-doc]
+    x, y, z
+):
+    """Example of a function with missing Google style parameter
+    documentation in the docstring
+
+    Args:
+        x: bla
+        z (int): bar
+
+    some other stuff
+    """
+    pass
```

Comment:
Wondering if we should do `print(x, y, z)` here and remove the disable. I'm not going to suggest for each commit as suggestion don't work in commit and this is an open question.

Reply:
I removed all `pass`'s